pkg/migrate/migrate.go

```diff
@@ -35,6 +35,10 @@ const IsDefaultStorageClassAnnotation = "storageclass.kubernetes.io/is-default-c
 
 const pvMigrateContainerName = "pvmigrate"
 
+const openEBSLocalProvisioner = "openebs.io/local"
+
+var isDestScLocalVolumeProvisioner bool
```

Comment:
ideally this would not be a global and it be passed as an argument through the stack

Reply:
I was resisting adding a global var but the refactor:time ratio was not in my favor. I'd like to clean this up in a subsequent update.

Reply:
I'm not entirely sure on what the benefit of having 'isDestScLocalVolumeProvisioner' at all is
why not just have that if statement be based on whether we know the hostname or not?

Reply:
@laverya The purpose of `isDestScLocalVolumeProvisioner` is to determine whether to add nodeAffinity info to the migration pod when migrating to local volume provisioner. Moreover, the nodeAffinity on the migration pod ensures that local volumes get created on the nodes where the original consumers were scheduled on.

Reply:
My point is "why not do this for ALL storage provisioners", if that makes sense

Reply:
I only scoped it for _local_ volume storage provisioners because it makes. For non-local(distributed) volumes, creating volumes that are "node aware" doesn't make sense.